spec/controllers/relationships_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'spec_helper'
+
+describe RelationshipsController do
+  let(:ryan) {Fabricate(:user)}
+  let(:laura) {Fabricate(:user)}
+  let(:ranger) {Fabricate(:user)}
+  let(:relationship) {Fabricate(:relationship, follower_id: ryan.id, leader_id: laura.id)}
+  let(:relationship2) {Fabricate(:relationship, follower_id: ranger.id, leader_id: laura.id)}
+  describe "GET index" do
+    
+    it "sets @relationships to the current user's following relationships" do
+      # ryan = Fabricate(:user)
+      set_current_user(ryan)
+      # laura = Fabricate(:user)
+      # relationship = Fabricate(:relationship, follower_id: ryan.id, leader_id: laura.id)
```

Comment:
I am not sure what is the value of these comments here. For code review I would recommend removing them.